### DISCUSSION TITLE
Unwrapping regions

### DISCUSSION THREAD

**Discussion by omarwater98** [gmtsar/gmtsar#440] [Q&A]
I can't found the file phasefilt.grd ?
the command:
gmt gridinfo phasefilt.grd

gives me this error 

ERROR: No module named gridinfo was found.  This could mean one of four cases:
  1. There actually is no such module; please check your spelling.
  2. You used a modern mode module name while running in GMT classic mode.
  3. Module exists in the GMT supplemental library, but the library could not be found.
  4. Module exists in a GMT custom library, but none was specified via GMT_CUSTOM_LIBS.
Shared libraries must be in standard system paths or set via environmental parameter LD_LIBRARY_PATH.

the values of minX/maxX/minY/maxY
have to changed each value of X, and Y for create the landmask_grd?

**Comment by Xiaohua-Eric-Xu** [comment]
I think it should be `grdinfo` instead of gridinfo.

**Reply by omarwater98** [reply]
The intf.in file only have to contain pairs where the master image is first on the pair?

**Reply by Xiaohua-Eric-Xu** [reply]
I am not sure what you mean exactly, but intf.in should contain all interferometric pairs you plan to produce after coregistration.

**Reply by omarwater98** [reply]
in the example of the document, TIMESERIES, the master image has in the pairs of the intf.in:

S1_20180508_ALL_F1:S1_20180514_ALL_F1
S1_20180508_ALL_F1:S1_20180526_ALL_F1
S1_20180508_ALL_F1:S1_20180607_ALL_F1
S1_20180508_ALL_F1:S1_20180520_ALL_F1

my question is, only the master image have to be first on the pair or if i have another pair like will dont posible make it

S1_20180607_ALL_F1:S1_20180514_ALL_F1

**Reply by Xiaohua-Eric-Xu** [reply]
> S1_20180508_ALL_F1:S1_20180514_ALL_F1
> S1_20180508_ALL_F1:S1_20180526_ALL_F1
> S1_20180508_ALL_F1:S1_20180607_ALL_F1
> S1_20180508_ALL_F1:S1_20180520_ALL_F1

This is not really the best example. You should be able to go ahead with `S1_20180607_ALL_F1:S1_20180514_ALL_F1` in your intf.in.

**Reply by omarwater98** [reply]
@Xiaohua-Eric-Xu  

surface [ERROR]: No datapoints inside region, aborting
surface [ERROR]: No datapoints inside region, aborting

i have the this error with the command

./preproc_batch_tops.csh data.in dem.grd 2 &> pbt_mode2.log

Only in the F3/raw directory.

**Reply by omarwater98** [reply]
surface [ERROR]: No datapoints inside region, aborting
surface [ERROR]: No datapoints inside region, aborting
[2]  + Exit 78                gmt surface atmp.xyz -bi3d -R0/23968/0/12192 -I8/4 -Gatmp.grd -T0.5 -N1000 -r
[1]  + Exit 78                gmt surface rtmp.xyz -bi3d -R0/23968/0/12192 -I8/4 -Grtmp.grd -T0.5 -N1000 -r
grdmath [ERROR]: Syntax error: rtmp.grd is not a number, operator or file name
grdmath [ERROR]: Syntax error: atmp.grd is not a number, operator or file name
Reading in range and azimuth shifts table...
make_s1a_tops [ERROR]: File r.grd not found
[Session make_s1a_tops (0)]: Error returned from GMT API: GMT_FILE_NOT_FOUND (16)
 cannot open range shift tables r.grd 

this appers in the pbt_mode2.log file

**Reply by Xiaohua-Eric-Xu** [reply]
These errors mean that your dem.grd does not really cover your F3 data.

**Reply by omarwater98** [reply]
it means that I don't need the F3, so i can continue with out F3?

**Reply by Xiaohua-Eric-Xu** [reply]
I believe so. But do note that if your DEM is smaller than your interferogram, radar phase will be valid only inside your selected region with DEM.

**Reply by omarwater98** [reply]
the radar phase? I mean I'm not going to notice the difference in level? or as?
the interferogram is practically all the image you download, could you cut the interferogram and work with a part?

**Reply by Xiaohua-Eric-Xu** [reply]
You can cut after processing.
